tests/Support/SupportStrTest.php

```diff
@@ -402,10 +402,10 @@ public function testConvertCase()
 
     public function testDedup()
```

Comment:
@taylorotwell, the method in the test should also be renamed from `testDedup` to `testDeduplicate` for the sake of standardised naming.

Reply:
I'll clean this up in another quick PR.